src/components/randomDrinkCards/RandomDrinkCards.tsx

```diff
@@ -7,11 +7,14 @@ type Drink = {
   strDrinkThumb: string;
 };
 
-const RandomDrinkCards: FC<Drink> = () => {
+const RandomDrinkCards: FC<Drink> = ({ strDrink, strDrinkThumb, idDrink }) => {
   return (
     <div className="drink-card">
-      <h2>Stuff will be here</h2>
-      {/* <img/> */}
+      <h2>{strDrink}</h2>
+      <img
+        src={strDrinkThumb}
+        alt={`Picture${idDrink} of a cocktail stirred to perfection`}
```

Comment:
Really great thought to include an `alt tag` for images to improve accessibility! Wondering if we should use the title of the drink interpolated here ( `strDrinkThumb` ) as well to be even more detailed on the picture description?